### PR TITLE
updated cdk.json for prod deployment

### PR DESCRIPTION
**Pull request by linglp**
PR Checklist:
[x] Describe and explain your intentions for this change
[x] Setup pre-commit and run the validators (info in README.md)

Updated cdk script for production deployment. This is related to: https://sagebionetworks.jira.com/browse/DCA-87

### CODE REVIEW

File: cdk.json [hunk 1]
```diff
@@ -38,6 +38,15 @@
...
       "ACM_CERT_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6",
       "VPC_CIDR": "10.255.73.0/24"
+    },
+    "prod": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/schematic:0.1.32-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "7080",
+      "COST_CENTER": "NO PROGRAM / 000000",
+      "STACK_NAME_PREFIX": "schematic",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6",
```
Comment (ConsoleCatzirl) [inline cdk.json]: This should refer to the certificate in the prod account, not the dev account. Are we deploying this into the dca-prod account?

**Review by brucehoff — APPROVED** [review]

**Review by zaro0508 — APPROVED** [review]

**Review by ConsoleCatzirl — CHANGES_REQUESTED** [review]

File: cdk.json (hunk 1 above)
Comment (linglp) [reply cdk.json]: Thanks for the feedback @jesusaurus . Do you know what is the certificate that I should use then? We are deploying this to schematic-prod account (if there's one).

Comment (ConsoleCatzirl) [reply cdk.json]: According to DCA-87 this is also going to the dca-prod account, so the correct value should be `arn:aws:acm:us-east-1:878654265857:certificate/d11fba3c-1957-48ba-9be0-8b1f460ee970`

Comment (linglp) [reply cdk.json]: Thank you! I just updated the certificate.

**Review by ConsoleCatzirl — APPROVED** [review]

**Pull request merged**